<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EscenaPolitica</class>
 <widget class="QDialog" name="EscenaPolitica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="bnOpcionC">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: rgba(0, 0, 0, 150); /* fondo negro translúcido */
    color: white;                         /* texto blanco */
    font-family: &quot;Segoe UI&quot;;              /* tipo de fuente */
    font-size: 9pt;    
  font-weight: bold;                     /* tamaño de texto */
    border-radius: 10px;                  /* bordes suaves */
    padding: 8px 15px;                    /* espacio interno */
    qproperty-alignment: 'AlignCenter';   /* centra el texto */
}
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCantidad">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 180);  /* fondo blanco semi-transparente */
    color: black;                                /* texto negro */
    font-family: &quot;Segoe UI&quot;;                     /* tipo de fuente */
    font-size: 12pt;                             /* tamaño de texto */
    font-weight: bold;                           /* texto más marcado */
    border-radius: 10px;                         /* bordes redondeados */
    padding: 8px 15px;                           /* espacio interno */
    qproperty-alignment: 'AlignCenter';          /* centra el texto */
}

</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bnOpcionA">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: rgba(0, 0, 0, 150); /* fondo negro translúcido */
    color: white;                         /* texto blanco */
    font-family: &quot;Segoe UI&quot;;              /* tipo de fuente */
    font-size: 9pt;     
  font-weight: bold;                    /* tamaño de texto */
    border-radius: 10px;                  /* bordes suaves */
    padding: 8px 15px;                    /* espacio interno */
    qproperty-alignment: 'AlignCenter';   /* centra el texto */
}
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewPolitica">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bnOpcionD">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: rgba(0, 0, 0, 150); /* fondo negro translúcido */
    color: white;                         /* texto blanco */
    font-family: &quot;Segoe UI&quot;;              /* tipo de fuente */
    font-size: 9pt;      
  font-weight: bold;                   /* tamaño de texto */
    border-radius: 10px;                  /* bordes suaves */
    padding: 8px 15px;                    /* espacio interno */
    qproperty-alignment: 'AlignCenter';   /* centra el texto */
}
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPregunta">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: rgba(0, 0, 0, 150); /* fondo negro translúcido */
    color: white;                         /* texto blanco */
    font-family: &quot;Segoe UI&quot;;              /* tipo de fuente */
    font-size: 14pt;                      /* tamaño de texto */
    font-weight: bold;                    /* texto más marcado */
    border-radius: 10px;                  /* bordes suaves */
    padding: 8px 15px;                    /* espacio interno */
    qproperty-alignment: 'AlignCenter';   /* centra el texto */
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTiempo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 180);  /* fondo blanco semi-transparente */
    color: black;                                /* texto negro */
    font-family: &quot;Segoe UI&quot;;                     /* tipo de fuente */
    font-size: 12pt;                             /* tamaño de texto */
    font-weight: bold;                           /* texto más marcado */
    border-radius: 10px;                         /* bordes redondeados */
    padding: 8px 15px;                           /* espacio interno */
    qproperty-alignment: 'AlignCenter';          /* centra el texto */
}

</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bnOpcionB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: rgba(0, 0, 0, 150); /* fondo negro translúcido */
    color: white;                         /* texto blanco */
    font-family: &quot;Segoe UI&quot;;              /* tipo de fuente */
    font-size: 9pt;    
  font-weight: bold;                    /* tamaño de texto */
    border-radius: 10px;                  /* bordes suaves */
    padding: 8px 15px;                    /* espacio interno */
    qproperty-alignment: 'AlignCenter';   /* centra el texto */
}
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVidrio3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbVidrio2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbVidrio1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbVidrio5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbVidrio4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sus">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>261</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsViewPolitica</zorder>
  <zorder>lbCantidad</zorder>
  <zorder>bnOpcionA</zorder>
  <zorder>bnOpcionD</zorder>
  <zorder>lbPregunta</zorder>
  <zorder>lbTiempo</zorder>
  <zorder>lbVidrio3</zorder>
  <zorder>lbVidrio1</zorder>
  <zorder>lbVidrio2</zorder>
  <zorder>lbVidrio4</zorder>
  <zorder>lbVidrio5</zorder>
  <zorder>sus</zorder>
  <zorder>bnOpcionB</zorder>
  <zorder>bnOpcionC</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
